### PR TITLE
remove references to "root site"

### PR DESCRIPTION
**Pull request by divyachapan**

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Enable-SPOCommSite.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ If the site URL input points to a site that does not exist, you will see the fol
 
 **Error case 4: Classic publishing feature is currently enabled**
 
-We do not support root site that currently have or have had in the past enabled the classic publishing features. If you see this error, your root site is not eligible for this feature enablement
+We do not support sites that currently have or have had in the past enabled the classic publishing features. If you see this error, your site is not eligible for this feature enablement. Please check [requirements for this feature](https://docs.microsoft.com/en-us/sharepoint/modernize-classic-team-site)
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Enable-SPOCommSite.md:70]:
```suggestion
We do not support sites that currently have or have had in the past enabled the classic publishing features. If you see this error, your site is not eligible for this feature enablement. Please check [requirements for this feature](https://docs.microsoft.com/sharepoint/modernize-classic-team-site)
```

**Comment by yogkumgit** [comment]
@divyachapan Please update this PR as per the review comments.

File: sharepoint/sharepoint-ps/sharepoint-online/Enable-SPOCommSite.md [hunk 2]
```diff
@@ -67,7 +67,7 @@ If the site URL input points to a site that does not exist, you will see the fol
 
 **Error case 4: Classic publishing feature is currently enabled**
 
-We do not support root site that currently have or have had in the past enabled the classic publishing features. If you see this error, your root site is not eligible for this feature enablement
+We do not support sites that currently have or have had in the past enabled the classic publishing features. If you see this error, your site is not eligible for this feature enablement. Please check [requirements for this feature](https://docs.microsoft.com/sharepoint/modernize-classic-team-site)
```
Comment (tseward) [inline sharepoint/sharepoint-ps/sharepoint-online/Enable-SPOCommSite.md:70]: Add a period to the end of the last sentence.

**Comment by yogkumgit** [comment]
@divyachapan Please implement the changes suggested by reviewers. Thanks

**Pull request merged**